assemblyline_v4_service/testing/helper.py

```diff
@@ -422,6 +422,7 @@ def _file_compare(ih: IssueHelper, f_type, original, new):
     def regenerate_results(self, save_files=False):
         for f in self.result_list():
             try:
+                print(f"Executing {f}")
```

Comment:
Unrelated to artifact handling, but useful when regenerating results with pytest